ui/components/multichain/app-header/app-header-unlocked-content.tsx

```diff
@@ -289,12 +289,10 @@ export const AppHeaderUnlockedContent = ({
             display={Display.Flex}
             justifyContent={JustifyContent.flexEnd}
             width={BlockSize.Full}
+            style={{ position: 'relative' }}
```

Comment:
Moved and fixed notification icon positioning on the larger menu icon.

<img width="75" height="80" alt="Screenshot 2025-08-20 at 22 08 50" src="https://github.com/user-attachments/assets/a11319df-1428-4b18-9718-65032015d26a" />